committees/backends.py

```diff
@@ -21,6 +21,10 @@ def has_perm(self, user_obj, perm, obj=None):
             content_type__app_label=app_label,
             codename=codename,
             group__associationgroup__members__in=[member],
+        ).exists() or Permission.objects.filter(
```

Comment:
I understand what you mean about depracting the link, but I'll keep it like this to ensure there won't suddenly be a conflict on the site that I might have missed. Another use of this approach is that it's easier to move board permissions to a different group.
That said, I do prefer redacting it in the long run, but I think that should be a seperate PR as depracting anything should not be done in a massive PR like this (to prevent any issues unexpected issues arising in working branches)